README.md

```diff
@@ -236,6 +237,17 @@ Example:
 $ touch trick.md
 ```
 
+### s. `find`
```

Comment:
Also place this command after [this](https://github.com/Idnan/bash-guide/pull/21/files#diff-04c6e90faac2675aa89e2176d2eec7d8R125) command.